mix.exs

```diff
@@ -20,15 +20,17 @@ defmodule Fennec.Mixfile do
   end
 
   def application do
-    [extra_applications: [:logger],
+    [extra_applications: [:logger, :runtime_tools],
```

Comment:
We use `:crypto` somewhere in the code, it needs to be added to `:extra_applications`.